vector/src/main/java/im/vector/app/features/MainActivity.kt

```diff
@@ -131,6 +141,8 @@ class MainActivity : VectorBaseActivity<ActivityMainBinding>(), UnlockedActivity
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        shortcutsHandler.updateShortcutsWithPreviousIntent()
```

Comment:
Should probably be better to put that in the ViewModel, but like many other things in this Activity... This would deserve a proper rework.